### PR TITLE
lazarsoma-FTS-59

### PR DESCRIPTION
**Pull request by lazarsoma**
Create Springfox documentation:

Dependencies
LevelupApplication: EnableSwagger2 annotation and apiDocket method

### CODE REVIEW

File: src/main/java/com/greenfoxacademy/levelup/controller/BadgeLibraryController.java [hunk 1]
```diff
@@ -4,7 +4,6 @@
 import com.greenfoxacademy.levelup.utility.Util;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
-import org.springframework.ui.Model;
```
Comment (hulfyback) [inline src/main/java/com/greenfoxacademy/levelup/controller/BadgeLibraryController.java]: I've got an error, it's missing this import

File: src/main/java/com/greenfoxacademy/levelup/model/User.java [hunk 2]
```diff
@@ -10,11 +10,15 @@
 import javax.persistence.MapsId;
 import javax.persistence.OneToMany;
 import lombok.AccessLevel;
+
```
Comment (hulfyback) [inline src/main/java/com/greenfoxacademy/levelup/model/User.java]: unnecessary empty line

**Review by hulfyback — CHANGES_REQUESTED** [review]
almost... :grinning:

**Review by hulfyback — APPROVED** [review]
It's OK, I guess :man_shrugging:

**Review by drexeverest — APPROVED** [review]

**Pull request merged**